server/controllers/notebook.js

```diff
@@ -137,4 +137,31 @@ exports.UpdateNotebook = function (req, res) {
   }
 
   Notebook.findById(notebookId, handleFindNotebook)
+}
+
+exports.GetStarredNotebooks = function (req, res){
```

Comment:
GET starred would be in the users model where we would have an array containing notedetails of both starred notebooks and notepages. 

Inside the notebook and notepage models we need a PUT request to update the starred boolean value.